internal/charmstore/addentity.go

```diff
@@ -513,11 +513,12 @@ func (s *Store) addEntity(entity *mongodoc.Entity) (err error) {
 		Write: perms,
 	}
 	baseEntity := &mongodoc.BaseEntity{
-		URL:         entity.BaseURL,
-		User:        entity.User,
-		Name:        entity.Name,
-		Public:      false,
-		ACLs:        acls,
+		URL:  entity.BaseURL,
+		User: entity.User,
+		Name: entity.Name,
+		ChannelACLs: map[mongodoc.Channel]mongodoc.ACL{
+			mongodoc.UnpublishedChannel: acls,
```

Comment:
Yes, definitely. But in this PR we're keeping things more directly equivalent to the old semantics to avoid changing too much at the same time. That means that we're translating ACLs to its Unpublished equivalent but no more.